include/apple2.h

```diff
@@ -151,20 +173,10 @@ extern unsigned char _dos_type;
 
 
 /* The file stream implementation and the POSIX I/O functions will use the
-** following struct to set the date and time stamp on files. This specificially
-** applies to the open and fopen functions.
+** following struct to set/get the date and time stamp on files. This specifically
+** applies to the open, fopen, stat and readdir functions.
```

Comment:
Is this a correct comment here? I think _datetime only applies to open and fopen.

Reply:
Uh indeed, that's a leftover from when I had put it on the struct.